SU-SWS/saa_victoria_subtheme
# ADAPT-525 Fixed heading 1, testimonial padding-bottom, and changed svg to png

# ALMOST READY FOR REVIEW

# Summary
- Fixed heading 1, testimonial padding-bottom, and changed svg to png

# Review By (Date)
- 07/27

# Urgency
- How critical is this PR?

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Review the site and confirm the following

- H1 font-size should use modular-typography(4) and be roughly `44px`
- Testimonial banner bottom padding should now use modular-spacing('padding-bottom', 6) and be roughly `9.5rem`
- SVG should now be PNG

# Associated Issues and/or People
- [ADAPT-525](https://stanfordits.atlassian.net/browse/ADAPT-525)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
